css-sizing-3/Overview.bs

```diff
@@ -370,7 +370,7 @@ Preferred Size Properties: the 'width' and 'height' properties</h4>
 	Name: width, height
 	Value: auto | <<length-percentage [0,∞]>> | min-content | max-content | <nobr>fit-content(<<length-percentage [0,∞]>>)</nobr> | <<width/calc-size()>>
 	Initial: auto
-	Applies to: all elements except <a>non-replaced</a> <a>inlines</a>
+	Applies to: all elements except <a>non-replaced</a> inline elements. 'width' cannot be applied to table rows, and row groups. 'height' cannot be applied to table columns, and column groups
```

Comment:
With the right `writing-mode`, `width` can indeed apply to table rows, and `height` to table columns and column groups.

```html
<!DOCTYPE html>
<table style="writing-mode: vertical-lr">
  <col style="height: 50px">
  <tr style="width: 100px">
    <td style="background: cyan"></td>
  </tr>
</table>
```

Reply:
You're right. But do we really want to put such a long sentence into 'Applies to'? Should we consider adding some examples below to illustrate these cases?

Reply:
No, I think it's better to leave "Applies to" as is. It would make sense for `inline-size` and `block-size`, but it would be a bit weird to have properties in a logical property group with different "Applies to". So maybe in prose.